diff -Naur a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2020-03-09 10:38:23.000000000 +0000
+++ b/CMakeLists.txt	2020-03-31 15:04:17.178115063 +0000
@@ -54,7 +54,7 @@
 # get sysrepo plugins directory
 if (NOT SRPD_PLUGINS_PATH)
     if (PKG_CONFIG_FOUND)
-        execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE} "--variable=SRPD_PLUGINS_PATH" "libsysrepo" OUTPUT_VARIABLE SRPD_PLUGINS_PATH)
+        execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE} "--variable=SRPD_PLUGINS_PATH" "sysrepo" OUTPUT_VARIABLE SRPD_PLUGINS_PATH)
         string(STRIP ${SRPD_PLUGINS_PATH} SRPD_PLUGINS_PATH)
     endif()
 endif()
