diff --git a/src/cb_streamid.c b/src/cb_streamid.c
index a8d14ff..9cf2395 100644
--- a/src/cb_streamid.c
+++ b/src/cb_streamid.c
@@ -594,7 +594,7 @@ int parse_streamid_per_port_per_id(sr_session_ctx_t *session, bool abort)
 			continue;
 		}
 
-		rc = sr_get_items(session, xpath, 0, &values, &count);
+		rc = sr_get_items(session, xpath, 0, 0, &values, &count);
 		if (rc == SR_ERR_NOT_FOUND) {
 			rc = SR_ERR_OK;
 			cur_node = cur_node->next;
diff --git a/src/common.c b/src/common.c
index 13e4ba6..32f247e 100644
--- a/src/common.c
+++ b/src/common.c
@@ -149,7 +149,7 @@ void print_config_iter(sr_session_ctx_t *session, const char *path)
 	if (!path || !session)
 		return;
 
-	rc = sr_get_items(session, path, 0, &values, &count);
+	rc = sr_get_items(session, path, 0, 0, &values, &count);
 	if (rc != SR_ERR_OK) {
 		printf("Error by sr_get_items: %s", sr_strerror(rc));
 		return;
diff --git a/src/qbu.c b/src/qbu.c
index 1d31108..0042642 100644
--- a/src/qbu.c
+++ b/src/qbu.c
@@ -141,7 +141,7 @@ int config_qbu_per_port(sr_session_ctx_t *session, char *path, bool abort,
 	char xpath[XPATH_MAX_LEN] = {0};
 	char err_msg[MSG_MAX_LEN] = {0};
 
-	rc = sr_get_items(session, path, 0, &values, &count);
+	rc = sr_get_items(session, path, 0, 0, &values, &count);
 	if (rc == SR_ERR_NOT_FOUND) {
 		/*
 		 * If can't find any item, we should check whether this
diff --git a/src/qbv.c b/src/qbv.c
index 79a2c76..3a48b33 100644
--- a/src/qbv.c
+++ b/src/qbv.c
@@ -309,7 +309,7 @@ int config_qbv_per_port(sr_session_ctx_t *session, char *path, bool abort,
 
 	init_qbv_memory(&qbvconf);
 
-	rc = sr_get_items(session, path, 0, &values, &count);
+	rc = sr_get_items(session, path, 0, 0, &values, &count);
 	if (rc == SR_ERR_NOT_FOUND) {
 		/*
 		 * If can't find any item, we should check whether this
diff --git a/src/qci_fm.c b/src/qci_fm.c
index a71d947..158feff 100644
--- a/src/qci_fm.c
+++ b/src/qci_fm.c
@@ -278,7 +278,7 @@ int parse_fm_per_port_per_id(sr_session_ctx_t *session, bool abort)
 			continue;
 		}
 
-		rc = sr_get_items(session, xpath, 0, &values, &count);
+		rc = sr_get_items(session, xpath, 0, 0, &values, &count);
 		if (rc == SR_ERR_NOT_FOUND) {
 			rc = SR_ERR_OK;
 			cur_node = cur_node->next;
diff --git a/src/qci_sf.c b/src/qci_sf.c
index c342051..647493a 100644
--- a/src/qci_sf.c
+++ b/src/qci_sf.c
@@ -282,7 +282,7 @@ int parse_sf_per_port_per_id(sr_session_ctx_t *session, bool abort)
 			continue;
 		}
 
-		rc = sr_get_items(session, xpath, 0, &values, &count);
+		rc = sr_get_items(session, xpath, 0, 0, &values, &count);
 		if (rc == SR_ERR_NOT_FOUND) {
 			rc = SR_ERR_OK;
 			cur_node = cur_node->next;
diff --git a/src/qci_sg.c b/src/qci_sg.c
index 496974a..e4b110b 100644
--- a/src/qci_sg.c
+++ b/src/qci_sg.c
@@ -401,7 +401,7 @@ int parse_sg_per_port_per_id(sr_session_ctx_t *session, bool abort)
 			continue;
 		}
 
-		rc = sr_get_items(session, xpath, 0, &values, &count);
+		rc = sr_get_items(session, xpath, 0, 0, &values, &count);
 		if (rc == SR_ERR_NOT_FOUND) {
 			rc = SR_ERR_OK;
 			cur_node = cur_node->next;
