diff -Naur a/scripts/model-install.sh b/scripts/model-install.sh
--- a/scripts/model-install.sh	2020-03-09 11:38:23.000000000 +0100
+++ b/scripts/model-install.sh	2020-04-04 17:18:44.387441621 +0200
@@ -8,7 +8,9 @@
 
 : ${SYSREPOCTL:=sysrepoctl}
 : ${SYSREPOCFG:=sysrepocfg}
-: ${SYSREPOCTL_ROOT_PERMS:=-o root:root -p 600}
+OWNER=$(id -un)
+GROUP=$(id -gn ${OWNER})
+SYSREPOCTL_ROOT_PERMS="-o ${OWNER} -g ${GROUP} -p 600"
 : ${YANG_DIR:=$local_path/../modules}
 
 is_yang_module_installed() {
@@ -22,7 +24,9 @@
 
     if ! is_yang_module_installed $module; then
         echo "- Installing module $module..."
-        $SYSREPOCTL -i -g ${YANG_DIR}/$module.yang $SYSREPOCTL_ROOT_PERMS
+        $SYSREPOCTL -i ${YANG_DIR}/$module.yang -s "${YANG_DIR}" -v 4
+        module_basename="${module%%@[-0-9]*}"
+        $SYSREPOCTL -c $module_basename $SYSREPOCTL_ROOT_PERMS
     else
         echo "- Module $module already installed."
     fi
@@ -34,24 +38,39 @@
 
     if ! $SYSREPOCTL -l | grep --count "^$module [^|]*|[^|]*|[^|]*|[^|]*|[^|]*|[^|]*|.* $feature.*$" > /dev/null; then
         echo "- Enabling feature $feature in $module..."
-        $SYSREPOCTL -m $module -e $feature
+        $SYSREPOCTL -c $module -e $feature
     else
         echo "- Feature $feature in $module already enabled."
     fi
 }
 
+wget -P "${YANG_DIR}" https://raw.githubusercontent.com/YangModels/yang/master/standard/ietf/RFC/ietf-interfaces@2014-05-08.yang
+wget -P "${YANG_DIR}" https://raw.githubusercontent.com/YangModels/yang/master/standard/ieee/draft/802.1/Qcw/ieee802-dot1q-types.yang
+wget -P "${YANG_DIR}" https://raw.githubusercontent.com/YangModels/yang/master/standard/ieee/draft/802.1/Qcw/ieee802-dot1q-preemption.yang
+wget -P "${YANG_DIR}" https://raw.githubusercontent.com/YangModels/yang/master/standard/ieee/draft/802.1/Qcw/ieee802-dot1q-sched.yang
+wget -P "${YANG_DIR}" https://raw.githubusercontent.com/YangModels/yang/master/standard/ietf/RFC/iana-if-type@2017-01-19.yang
+wget -P "${YANG_DIR}" https://raw.githubusercontent.com/YangModels/yang/master/standard/ieee/published/802.1/ieee802-dot1q-bridge.yang
+wget -P "${YANG_DIR}" https://raw.githubusercontent.com/YangModels/yang/master/standard/ietf/RFC/ietf-yang-types@2013-07-15.yang
+# mv "${YANG_DIR}/ietf-yang-types@2013-07-15.yang" "${YANG_DIR}/ietf-yang-types.yang"
+wget -P "${YANG_DIR}" https://raw.githubusercontent.com/YangModels/yang/master/standard/ieee/draft/802.1/Qcw/ieee802-types.yang
+wget -P "${YANG_DIR}" https://raw.githubusercontent.com/YangModels/yang/master/standard/ietf/RFC/ietf-inet-types@2013-07-15.yang
+# mv "${YANG_DIR}/ietf-inet-types@2013-07-15.yang" "${YANG_DIR}/ietf-inet-types.yang"
+wget -P "${YANG_DIR}" https://raw.githubusercontent.com/YangModels/yang/master/standard/ieee/draft/802.1/Qcr/ieee802-dot1q-stream-filters-gates.yang
+wget -P "${YANG_DIR}" https://raw.githubusercontent.com/YangModels/yang/master/standard/ieee/draft/802.1/Qcw/ieee802-dot1q-psfp.yang
+
+install_yang_module iana-if-type@2017-01-19
+
 install_yang_module ietf-interfaces@2014-05-08
 install_yang_module ieee802-dot1q-types
+
 install_yang_module ieee802-dot1q-preemption
 enable_yang_module_feature ieee802-dot1q-preemption frame-preemption
 
 install_yang_module ieee802-dot1q-sched
 enable_yang_module_feature ieee802-dot1q-sched scheduled-traffic
 
-install_yang_module iana-if-type@2017-01-19
-
 install_yang_module ieee802-dot1q-bridge
-install_yang_module ietf-yang-types
+install_yang_module ietf-yang-types@2013-07-15
 install_yang_module ieee802-types
 install_yang_module ietf-inet-types@2013-07-15
 install_yang_module ieee802-dot1q-stream-filters-gates
